packages/client/src/containers/CollectionsPage/CollectionsPage.Container.js

```diff
@@ -1,12 +1,91 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import ProductList from '../../components/ProductList/ProductList.component';
-import DropDownViewList from '../../components/ProductListDropDown/DropDownViewList/DropDownViewList.component';
+import './CollectionsPage.style.css';
+import DropDownView from '../../components/CategoriesListDropDown/CategoriesListDropDown.component';
+import { apiURL } from '../../apiURL';
 
 export const CollectionsPage = () => {
+  const [products, setProducts] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [onSelectedCategory, setOnSelectedCategory] = useState('');
+  const [onSelectedSortBy, setOnSelectedSortBy] = useState('');
+  // const [onSelectedAllFilter, setOnSelectedAllFilter] = useState('');
+
+  const fetchProducts = () => {
+    setIsLoading(true);
+    fetch(`${apiURL()}/products/`)
+      .then((res) => res.json())
+      .then((item) => {
+        setProducts(item);
+      })
+      .then(() => {
+        setIsLoading(false);
+      });
+  };
+  useEffect(() => {
+    fetchProducts();
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    const fetchProducts = () => {
      setIsLoading(true);
      fetch(`${apiURL()}/products`)
        .then((res) => res.json())
        .then((item) => {
          setProducts(item);
          setIsLoading(false);
        });
    };
    
    fetchProducts();
  }, []);
```

Prefer to put the logic of the effect inside of the effect so it's clear that this happens as an effect.